### PR TITLE
(SIMP-2971) Add $package_ensure

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-2971 #comment update aide

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-aide/pipelines/32605614

**Comment by jeefberkey** [comment]
:heavy_check_mark: https://gitlab.com/simp/pupmod-simp-aide/pipelines/32694933

File: spec/acceptance/suites/default/class_spec.rb [hunk 1]
```diff
@@ -73,18 +72,19 @@ class { 'aide': }
...
+        'simp_options::logrotate' => true,
+        'aide::syslog_format'     => true,
+        'auditd::enable'          => false,
+       }}
 
       it 'should work with no errors' do
         set_hieradata_on(host, hieradata)
         apply_manifest_on(host, manifest, :catch_failures => true)
+        # rsyslog changes require a second run
+        apply_manifest_on(host, manifest, :catch_failures => true)
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/default/class_spec.rb]: The problem can probably be solved by rsyslog::install::ensure = 'installed'.

**Comment by jeefberkey** [comment]
It looks like making the same change for simp/rsyslog should fix the acceptance tests in a better way, so I'm going to WIP this until that one gets through.

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-aide/pipelines/33010840

**Pull request merged**